enterprise/server/backends/pebble_cache/pebble_cache.go

```diff
@@ -2744,6 +2959,8 @@ func (e *partitionEvictor) deleteFile(key filestore.PebbleKey, version filestore
 		}
 	case storageMetadata.GetInlineMetadata() != nil:
 		break
+	case storageMetadata.GetChunkedMetadata() != nil:
+		break
```

Comment:
Do we need to skip updating the partition size in this case?

Reply:
I set the storedSizeBytes in the filemetadata is zero if it's a file level entry that include chunked metadata